code/exercises/word-count/test.ss

```diff
@@ -0,0 +1,37 @@
+(define (parse-test test)
+  `(lambda ()
+     (test-success ,(lookup 'description test)
+		   (lambda (result expected)
+		     (and (= (length expected)
+			     (hashtable-size result))
+			  (fold-left (lambda (count-agrees w.c)
+				       (and count-agrees
+					    (= (cdr w.c)
+					       (hashtable-ref result
+							      (car w.c)
+							      0))))
+				     #t
+				     expected)))
```

Comment:
Actually, this requires students to implement the solution using hashtables. I should redo the equality test to allow for association lists, too.